docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateAccess/templateAccess.rst

```diff
@@ -1,54 +1,70 @@
 
-In order set access to template,user needs  go to tab "Template" and click on "Template Access".
+How to modify template access level
+===================================
 
-.. image:: picTemplateAccess/Screenshot_700.png
+In order to set access level of template you need to go to "Template" tab and click on "Template Access".
+
+.. image:: picTemplateAccess/TemplateAccess_1.png
    :width: 400
    :align: center
 
-User can grant access to specific mailboxes. Next, you need to select edit access.
+You can grant access to specific mailboxes. After selecting this option you will need to click on edit button.
 
-.. image:: picTemplateAccess/Screenshot_76.png
+.. image:: picTemplateAccess/TemplateAccess_2.png
    :width: 400
    :align: center
 
-In the modal, enter: mailbox name, mailbox uuid, alias, or contact name.
+In the modal window you can enter mailbox name, mailbox uuid, alias, or contact name.
 
-.. image:: picTemplateAccess/Screenshot_72.png
+.. image:: picTemplateAccess/TemplateAccess_3.png
    :width: 400
    :align: center
 
-When the mailbox is added, click "Save".
+When mailbox is added click "Save".
 
-.. image:: picTemplateAccess/Screenshot_73.png
+.. image:: picTemplateAccess/TemplateAccess_4.png
    :width: 400
    :align: center
 
 Save сhanges.
 
-.. image:: picTemplateAccess/Screenshot_74.png
+.. image:: picTemplateAccess/TemplateAccess_5.png
+   :width: 400
+   :align: center
+
+After saving еру template it will be available for mailboxes specified by you in the folder "Shared with me".
+
+.. image:: picTemplateAccess/TemplateAccess_6.png
    :width: 400
    :align: center
+   
+Additional sharing access configurations
+========================================
 
-After saving template, it will be available for mailboxes specified in the access. And be in the folder: "Shared with me".
+At the bottom of template tab of envelope menu you will find "Do not send files by email" attribute.
 
-.. image:: picTemplateAccess/Screenshot_725.png
+.. image:: picTemplateAccess/TemplateAccess_7.png
    :width: 400
    :align: center
+   
+If you will enable it and save the template, all envelopes created from it will not contain any files from envelope in notification emails. Also such envelopes can not be shared with other mailboxes.
 
+.. warning::
+   If you will enable this toggle, it wouldn't be possible to turn in off in the envelopes. If it is disabled on template — it can be modified in the envelopes.
```

Comment:
"...to turn in off in the envelopes..." I suppose it is also typo in phrase "to turn in off". Probably it should be "to turn it off"